pliers/extractors/text.py

```diff
@@ -402,3 +400,47 @@ def _extract(self, stim):
 
         return ExtractorResult(features_list, stim, self,
                                features=self.features, orders=order_list)
+
+
+class WordCounterExtractor(BatchTransformerMixin, TextExtractor):
+
+    ''' Extracts number of times each unique word has occurred within text
+    
+    Args:
+        lemmatize(bool): specifies if words are to be lemmatized before being counted. 
+        log_scale(bool): specifies if count values are to be returned in log scale (defaults to False)
+        '''
+
+    _log_attributes = ('lemmatize','log_scale',)
+    _batch_size = sys.maxsize
+    
+    def __init__(self, lemmatize=None, log_scale=False):
+        
+        self.lemmatize = lemmatize
+        self.log_scale = log_scale
+        self.features = ['log_word_count'] if self.log_scale else ['word_count']
+        super(WordCounterExtractor, self).__init__()
+        
+    @requires_nltk_corpus
+    def _extract(self, stims):
+        
+        tokens = [s.text.lower() for s in stims]
```

Comment:
Maybe this should be an argument to the Extract init, though? You could potentially want counting to be case sensitive